ipad/ViewControllers/Shift/ShiftViewController.swift

```diff
@@ -343,6 +361,17 @@ class ShiftViewController: BaseViewController {
                 }
             }
         }
+
+        // Check for invalid inspections
+        let invalidInspections = model.inspections.filter { !$0.formDidValidate }
+        if !invalidInspections.isEmpty {
+            message = "\(message)\n\(counter)- One or more inspections contain validation errors. Please review each inspection."
+            counter += 1
+        }
+
+        if !message.isEmpty {
```

Comment:
This block may be redundant and removable with our other checks in place, however I will have to confirm